projects/addon-doc/src/interfaces/code-editor.ts

```diff
@@ -1,4 +1,5 @@
 export interface TuiCodeEditor {
     readonly name: string;
+    readonly icon?: string | null;
```

Comment:
What do your think about ?
```suggestion
    readonly icon?: PolymorpheusContent;
```

And default value:
```ts
Edit on {{ codeEditor!.name }}
```
(for backward compatibility)

Reply:
This gives more flexibility and allows to do anything: text + icon, only icon, text for desktop and icon for mobile etc.

Reply:
fixed

Reply:
Nope, let me explain again

Reply:
```html
<ng-container *polymorpheusOutlet="codeEditor?.editActionContent as text">
        {{ text }}
</ng-container>
```

and default value of `editActionContent` – `"Edit on ${codeEditor!.name}"`

Reply:
And for Taiga UI it will be
```html
const EDIT_STACKBLITZ_COMPONENT =  ... // Edit + <tui-svg src="assets/icons/stackblitz.svg">
```

[...]

```ts
export class TuiStackblitzService implements TuiCodeEditor {
    editActionContent = EDIT_STACKBLITZ_COMPONENT;
}
```

Reply:
(`editActionContent` is not the best naming, we should better one)